src/commands/new.ts

```diff
@@ -575,6 +575,14 @@ module.exports = {
       // #region Run Packager Install
       // pnpm/yarn/npm/bun install it
 
+      // fix .npmrc if using pnpm
+      if (packagerName === "pnpm") {
+        // append `node-linker=hoisted` to .npmrc
+        const npmrcPath = path(targetPath, ".npmrc")
+        const npmrcContents = read(npmrcPath)
+        write(npmrcPath, `${npmrcContents}${EOL}node-linker=hoisted${EOL}`)
```

Comment:
FYI, gluegun has a built-in patching library that [makes this a little simpler](https://github.com/infinitered/gluegun/blob/master/docs/toolbox-patching.md#append).

To keep the style elsewhere in this file don't use this, but as an (untested) example:

```suggestion
        const npmrcPath = path(targetPath, ".npmrc")
        await patching.append(npmrcPath, `${EOL}node-linker=hoisted${EOL}`)
```